content/en/docs/setup/install/multicluster/before-you-begin/index.md

```diff
@@ -18,7 +18,7 @@ In addition, review the requirements and perform the initial steps below.
 ### Cluster
 
 This guide requires that you have two Kubernetes clusters with any of the
-[supported Kubernetes versions](/docs/setup/platform-setup).
+supported Kubernetes versions: {{< supported_kubernetes_versions >}}.
```

Comment:
I had just noticed that the Getting Started page could also use something like this. Can you change that one to something like this:
```
These steps require you to have a {{< gloss >}}cluster{{< /gloss >}} running a
compatible version of Kubernetes ({{< supported_kubernetes_versions >}}). You can use ...
```

Reply:
Updated.